### ISSUE TITLE
About dice_loss

### ISSUE THREAD

**Issue by PatrickWilliams44** [Minerva-jiezhao/Pytorch-Segmentation-multi-models#5]
Hi Minerva-J! My outputs.shape = [2, 3, 224, 224], targets_x.shape=[2, 224, 224], I want to use dice_loss as the loss function, it requires the input parameter shape to be [B, H, W], how should I modify it?
Looking forward to your early reply! Thank you a lot.
